sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClientBuilderExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static class WebPubSubServiceClientBuilderExtensions
         public static IAzureClientBuilder<WebPubSubServiceClient, WebPubSubServiceClientOptions> AddWebPubSubServiceClient<TBuilder>(this TBuilder builder, string connectionString, string hub)
             where TBuilder : IAzureClientFactoryBuilder
         {
-            return builder.RegisterClientFactory<WebPubSubServiceClient, WebPubSubServiceClientOptions>(options => new WebPubSubServiceClient(connectionString, hub, options));
+            return builder.RegisterClientFactory<WebPubSubServiceClient, WebPubSubServiceClientOptions>(options => new WebPubSubServiceClient(hub, connectionString, options));
```

Comment:
I'm surprised that "connection string" doesn't come first here.  It feels very similar to endpoint to me and we usually put that first.  We don't have anything prescriptive in the guidelines about how to order these parameters, should we?  (Note that BlobContainerClient's .ctor that uses a connection string and container name are ordered connection string first, container name second)

Reply:
Also it looks like `WebPubSubServiceClient` still expects connectionString, hubName, so is this even right?